### PR TITLE
Feature save entity version two

### PR DESCRIPTION
**Pull request by bogdansoft**
save & persist methods new implementation

### CODE REVIEW

**Comment by bogdansoft** [comment]
save() & persist() guys

File: src/main/java/com/ormanager/client/entity/Book.java [hunk 1]
```diff
@@ -25,6 +24,17 @@ public class Book {
     @NonNull
     private LocalDate publishedAt;
 
-    @ManyToOne(columnName = "publisher_id")
-    private Publisher publisher = null;
+    /*public Book(Long id, @NonNull String title, @NonNull LocalDate publishedAt) {
```
Comment (olhakad) [inline src/main/java/com/ormanager/client/entity/Book.java:27]: please remove commented code or it is needed for the next commit?

File: src/main/resources/create.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+create table if not exists book
```
Comment (olhakad) [inline src/main/resources/create.sql:1]: Can ddl schema be created based on registered classes?

**Comment by bogdansoft** [comment]
new pull request created with code fixes

**Review by olhakad — APPROVED** [review]

**Pull request merged**